[2023-08-29T08:14:35.287+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=dataset_consumes_1/run_id=manual__2023-08-29T08:14:31.663359+00:00/task_id=consuming_1 permission to 509
[2023-08-29T08:14:35.289+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=dataset_consumes_1/run_id=manual__2023-08-29T08:14:31.663359+00:00/task_id=consuming_1 permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dataset_consumes_1/run_id=manual__2023-08-29T08:14:31.663359+00:00/task_id=consuming_1'
[2023-08-29T08:14:35.546+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=dataset_consumes_1/run_id=manual__2023-08-29T08:14:31.663359+00:00/task_id=consuming_1 permission to 509
[2023-08-29T08:14:35.553+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=dataset_consumes_1/run_id=manual__2023-08-29T08:14:31.663359+00:00/task_id=consuming_1 permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dataset_consumes_1/run_id=manual__2023-08-29T08:14:31.663359+00:00/task_id=consuming_1'
[2023-08-29T08:14:35.691+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dataset_consumes_1.consuming_1 manual__2023-08-29T08:14:31.663359+00:00 [queued]>
[2023-08-29T08:14:35.708+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dataset_consumes_1.consuming_1 manual__2023-08-29T08:14:31.663359+00:00 [queued]>
[2023-08-29T08:14:35.709+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-08-29T08:14:35.886+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): consuming_1> on 2023-08-29 08:14:31.663359+00:00
[2023-08-29T08:14:35.896+0000] {standard_task_runner.py:57} INFO - Started process 9600 to run task
[2023-08-29T08:14:35.904+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dataset_consumes_1', 'consuming_1', 'manual__2023-08-29T08:14:31.663359+00:00', '--job-id', '19', '--raw', '--subdir', '/home/***/.local/lib/python3.8/site-packages/***/example_dags/example_datasets.py', '--cfg-path', '/tmp/tmplpkq0l8v']
[2023-08-29T08:14:35.910+0000] {standard_task_runner.py:85} INFO - Job 19: Subtask consuming_1
[2023-08-29T08:14:36.052+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=dataset_consumes_1/run_id=manual__2023-08-29T08:14:31.663359+00:00/task_id=consuming_1 permission to 509
[2023-08-29T08:14:36.053+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=dataset_consumes_1/run_id=manual__2023-08-29T08:14:31.663359+00:00/task_id=consuming_1 permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dataset_consumes_1/run_id=manual__2023-08-29T08:14:31.663359+00:00/task_id=consuming_1'
[2023-08-29T08:14:36.056+0000] {task_command.py:415} INFO - Running <TaskInstance: dataset_consumes_1.consuming_1 manual__2023-08-29T08:14:31.663359+00:00 [running]> on host c1ed3c1aff26
[2023-08-29T08:14:36.417+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dataset_consumes_1' AIRFLOW_CTX_TASK_ID='consuming_1' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T08:14:31.663359+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-08-29T08:14:31.663359+00:00'
[2023-08-29T08:14:36.434+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-29T08:14:36.436+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'sleep 5']
[2023-08-29T08:14:36.460+0000] {subprocess.py:86} INFO - Output:
[2023-08-29T08:14:41.708+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-29T08:14:41.897+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/models/baseoperator.py:1472 RemovedInAirflow3Warning: Passing 'execution_date' to 'TaskInstance.xcom_push()' is deprecated.
[2023-08-29T08:14:41.988+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dataset_consumes_1, task_id=consuming_1, execution_date=20230829T081431, start_date=20230829T081435, end_date=20230829T081441
[2023-08-29T08:14:42.270+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-29T08:14:42.309+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
